src/arviz_plots/backend/__init__.py

```diff
@@ -99,6 +99,29 @@ def create_plotting_grid(
 
 
 # "geoms"
+def hist(x, height, target, *, width=1.6, bottom=None, align="center", color=None):
+    """Interface for a hist plot.
+
+    Parameters
```

Comment:
We haven't been completely consistent adding the docstrings to this module, but the idea is for the documentation here so define the behaviour **all** backends have to comply with. It also isn't a general plotting backend interface but a common interface _for our own plots to use_ which means we can be quite restrictive about the features.

For example, I would only accept array_like for x, height, width... I don't think the extra edge case of floats would be used anywhere. It also means the descriptions should not depend on any backend.